### PR TITLE
Bug 1135806 - making sure utility tray and its contents are not visible ...

### PR DESCRIPTION
**Pull request by yzen**
...to screen reader when hidden off screen.

### CODE REVIEW

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2f4ef22036ca)

File: apps/system/js/utility_tray.js [hunk 1]
```diff
@@ -251,12 +251,14 @@ window.UtilityTray = {
...
 
       case 'statusbarwheel':
+        this.overlay.classList.add('visible');
         this.show();
         break;
       case 'wheel':
         if (evt.deltaMode === evt.DOM_DELTA_PAGE && evt.deltaY &&
           evt.deltaY > 0) {
           this.hide(true);
+          this.overlay.classList.remove('visible');
```
Comment (alivedise) [inline apps/system/js/utility_tray.js]: Could we put this in hide()?

File: apps/system/js/utility_tray.js [hunk 2]
```diff
@@ -251,12 +251,14 @@ window.UtilityTray = {
         break;
 
       case 'statusbarwheel':
+        this.overlay.classList.add('visible');
```
Comment (alivedise) [inline apps/system/js/utility_tray.js]: Could we put this in show()?

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=82f532649b3f)

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8869c66e2179)

**Pull request merged**